dev/com.ibm.ws.transport.http/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -53,7 +53,10 @@ endpoint.httpsPort=Secure port
 endpoint.httpsPort.desc=The port used for client HTTP requests secured with SSL (https). Use -1 to disable this port.
 
 endpoint.virtualHost=Virtual host
-endpoint.virtualHost.desc=ID of a configured virtual host. All endpoints are associated with the 'default_host' virtual host by default. 
+endpoint.virtualHost.desc=ID of a configured virtual host. All endpoints are associated with the 'default_host' virtual host by default.
+
+endpoint.protocolVersion=Protocol version
+endpoint.protocolVersion.desc=When Servlet 4.0 is enabled as a feature, set this attribute to http/1.1 to disable HTTP/2 processing for the ports that were defined for the httpEndpoint element. When Servlet 3.1 is enabled as a feature, set this attribute to http/2 to enable HTTP/2 processing for the ports that are defined for the httpEndpoint element.
```

Comment:
Change
"When Servlet 4.0 API is enabled as a feature, set this attribute to http/1.1 to disable HTTP/2 processing for the ports that were defined for the httpEndpoint element. When Servlet 3.1 API is enabled as a feature, set this attribute to http/2 to enable HTTP/2 processing for the ports that are defined for the httpEndpoint element."
to
"When the Servlet 4.0 API is enabled as a feature, set this attribute to http/1.1 to disable HTTP/2 processing for the ports that were defined for the httpEndpoint element. When the Servlet 3.1 API is enabled as a feature, set this attribute to http/2 to enable HTTP/2 processing for the ports that are defined for the httpEndpoint element."